run-ci-tests.sh

```diff
@@ -24,7 +24,11 @@ case $TEST_TYPE in
     "Unit tests" )
       echo "Running unit tests";
       npm run test:unit || exit 1
-      npm run test ;;
+      npm run test || exit 1
```

Comment:
Can you add !#/bin/bash and set -e to the top of this script so you don’t need to think about the || exits everywhere? One of these days we should lint for this

Reply:
I did that for the snarkyjs build script and for some reason it doesn't work the way I expected (script didn't stop on an error), so I came to distrust `set -e` :D maybe I just did something wrong, but I definitely don't want to risk test failures getting through. I'll try it out here though with an artificial test failure!

Reply:
it works now, I tested it out, removed all the `exit 1`s